styles/helpers/inputs.css

```diff
@@ -140,4 +140,56 @@ title: Form
       @mixin input var(--color-input-background), var(--color-body), var(--color-input-disabled), var(--color-input-error), var(--color-input-error), var(--color-input-error);
     }
   }
+
+  /*---
+  section: Forms
+  title: Input group
+
+  ```example:html
+  <form class="form">
+    <div class="input-group">
+      <input type="text" placeholder="Enter your email">
+      <button class="input-group-btn button">Sign Up</button>
+    </div>
+  </form>
+  ```
+  */
+  & .input-group {
+    font-size: 0;
+
+    & input[type="text"],
+    & input[type="email"],
+    & input[type="password"],
+    & input[type="url"],
+    & input[type="tel"],
+    & input[type="search"],
+    & input[type="number"],
+    & input[type="date"],
+    & input[type="datetime"],
+    & input[type="datetime-local"],
+    & input[type="month"],
+    & input[type="week"],
+    & input[type="time"] {
+      display: inline-block;
+      width: inherit;
+      height: 42px;
+      vertical-align: middle;
+      box-shadow: none;
+      margin-right: -1px;
+      padding-right: 14px;
+      padding-left: 14px;
+      border-radius: var(--border-radius) 0 0 var(--border-radius);
```

Comment:
The one where you changed this variable

On Fri, Sep 9, 2016, 4:40 PM Newton <notifications@github.com> wrote:

> In styles/helpers/inputs.css
> <https://github.com/firstandthird/clientkit/pull/103#discussion_r78263978>
> :
>
> > +    & input[type="number"],
> > +    & input[type="date"],
> > +    & input[type="datetime"],
> > +    & input[type="datetime-local"],
> > +    & input[type="month"],
> > +    & input[type="week"],
> > +    & input[type="time"] {
> > +      display: inline-block;
> > +      width: inherit;
> > +      height: 42px;
> > +      vertical-align: middle;
> > +      box-shadow: none;
> > +      margin-right: -1px;
> > +      padding-right: 14px;
> > +      padding-left: 14px;
> > +      border-radius: var(--border-radius) 0 0 var(--border-radius);
>
> what PR?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/firstandthird/clientkit/pull/103/files/196e6cfabb2131581368d385302fb569b9fbf574#r78263978>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AABtIDfK2ry4aNrYplP1O3rW5GvHPtdbks5qoe5fgaJpZM4J0QKD>
> .
>